shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-route/src/main/java/org/apache/shardingsphere/sharding/route/engine/validator/ShardingStatementValidatorFactory.java

```diff
@@ -66,6 +73,25 @@
         if (sqlStatement instanceof CreateProcedureStatement) {
             return Optional.of(new ShardingCreateProcedureStatementValidator());
         }
+        if (sqlStatement instanceof CreateViewStatement) {
+            return Optional.of(new ShardingCreateViewStatementValidator());
+        }
+        if (sqlStatement instanceof AlterViewStatement) {
```

Comment:
If `Create view` SQL can work well for `single table` currently, do you think `ShardingUnconfiguredTablesRoutingEngine` will be a better match?

Reply:
> If `Create view` SQL can work well for `single table` currently, do you think `ShardingUnconfiguredTablesRoutingEngine` will be a better match?

@tristaZero `ShardingUnconfiguredTablesRoutingEngine` can meet the needs of creating views on a single data source. I think it is a better choice. 👍